sprintf/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('jest').Config} */
+const config = {
+  // We only test JS files once compiled with TypeScript
+  moduleFileExtensions: ["js"],
+  coverageDirectory: "coverage",
+  collectCoverage: true,
+  // If this is set to default (babel) rather than v8, tests fail with the edge
+  // runtime and the error "EvalError: Code generation from strings disallowed
+  // for this context". Tracking in
+  // https://github.com/vercel/edge-runtime/issues/250
+  coverageProvider: "v8",
+  verbose: true,
+  testEnvironment: "node",
```

Comment:
Is it worth testing in the different runtimes we support? E.g. https://bun.sh/docs/cli/test

Reply:
Not really. I don't want to spend further time right now to figure out how to get Bun into the CI and run different sets of tests in the monorepo (ref #769).

There are no node imports but we don't want jest running in jsdom so this is just the easiest with our current harness.